### PR TITLE
[breaking] Adding missing type tag for jsonrpc view enums

### PR DESCRIPTION
**Pull request by thefallentree**
make it consistent with rest of the api

### CODE REVIEW

**Comment by xli** [comment]
2 things, could you update API-CHANGELOG.md and add test to test_json_rpc_protocol for the one does not covered yet.

**Comment by xli** [comment]
also update the type_account.md and type_transaction.md document

**Comment by thefallentree** [comment]
updated PTAL

File: json-rpc/docs/type_account.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ A Libra account.
...
-| role                              | string or object               | One of the following values:                                                                |
-|                                   |                                |   - "unknown"                                                                               |
-|                                   |                                |   - { "designated_dealer" => [DesignatedDealerRole](#type_designateddealerrole) }           |
-|                                   |                                |   - { "parent_vasp" => [ParentVASPRole](#type_parent_vasp) }                                |
-|                                   |                                |   - { "child_vasp" => [ChildVASPRole](#type_childvasprole) }                                |
+| role                              | object                         | One of the following values:                                                                |
+|                                   |                                |   - [UnknownRole](#type_unknown_vasp) }                                |
```
[the remark this answers is not in the document]
Comment (thefallentree) [reply json-rpc/docs/type_account.md]: fixed.

File: json-rpc/docs/type_account.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ A Libra account.
...
-|                                   |                                |   - { "designated_dealer" => [DesignatedDealerRole](#type_designateddealerrole) }           |
-|                                   |                                |   - { "parent_vasp" => [ParentVASPRole](#type_parent_vasp) }                                |
-|                                   |                                |   - { "child_vasp" => [ChildVASPRole](#type_childvasprole) }                                |
+| role                              | object                         | One of the following type:                                                                |
+|                                   |                                |   - [UnknownRole](#type_unknownrole) |
+|                                   |                                |   - [ParentVASPRole](#type_parentvasprole) |
+|                                   |                                |   - [ChildVASPRole](#type_childvasprole) |
+|                                   |                                |   - [DesignatedDealerRole](#type_designateddealerrole) |
```
Comment (xli) [inline json-rpc/docs/type_account.md:24]: it was my mistake, the underscore after type "_" should be dash "-"

**Comment by xli** [comment]
may also update method_get_account.md and method_get_account_transaction.md, method_get_account_transactions.md, method_get_transactions.md example responses.

copy the curl command and get response should work. 

looks like we should have a better way to update these example output.

File: json-rpc/docs/type_account.md (hunk 2 above)
Comment (thefallentree) [reply json-rpc/docs/type_account.md]: yeah, fixed.

**Comment by andll** [comment]
The failure is KP fixed in https://github.com/libra/libra/pull/5531

**Pull request closed without merging**